libs/stats/odc/stats/_pq_bitmask.py

```diff
@@ -0,0 +1,131 @@
+"""
+USGS Landsat pixel quality
+"""
+from functools import partial
+from typing import List, Optional, Tuple
+
+import dask.array as da
+import xarray as xr
+
+from odc.algo import mask_cleanup, keep_good_only
+from odc.algo._masking import _xr_fuse, _first_valid_np, _fuse_or_np
+from odc.algo.io import load_with_native_transform
+from odc.stats.model import Task
+
+from .model import StatsPluginInterface
+from . import _plugins
+
+
+class StatsPQLSBitmask(StatsPluginInterface):
+    NAME = "pc_ls_bitmask"
+    SHORT_NAME = NAME
+    VERSION = '0.0.1'
+    PRODUCT_FAMILY = "pixel_quality_statistics"
+
+    def __init__(
+            self,
+            pq_band: str = "QA_PIXEL",
+            aerosol_band: str = None,
+            filters: Optional[List[Tuple[int, int]]] = [(1, 2)],
+            resampling: str = "nearest",
+    ):
+        self.filters = filters
+        self.pq_band = pq_band
+        self.aerosol_band = aerosol_band
+        self.resampling = resampling
+
+    @property
+    def measurements(self) -> Tuple[str, ...]:
+        band = [
+            "total",
+            "clear",
+            *[f"clear_{r1:d}_{r2:d}" for (r1, r2) in self.filters],
+        ]
+        if self.aerosol_band is not None:
+            band.append("clear_aerosol")
+
+        return tuple(band)
+
+    def input_data(self, task: Task) -> xr.Dataset:
+        chunks = {"y": -1, "x": -1}
+        bands = [self.pq_band]
+        if self.aerosol_band is not None:
+            bands.append(self.aerosol_band)
```

Comment:
You don't need to do that again...

Reply:
This is input bands - `QA_PIXEL` and `SR_QA_AEROSOL` - for measurements

Reply:
Oh, right sorry!